### PR TITLE
Use gettext .po files, and translate everything

### PR DESCRIPTION
**Pull request by ossman**
Partial translations are worse than no translations IMO. So this sorts out having the HTML translated as well.

It also switches to using gettext files and tools for translations. This makes it much easier to work with translations, and also allows us to have external parties do translations in a much more straight forward way.

### CODE REVIEW

**Comment by samhed** [comment]
Really nice!

Is there any reason to why you are keeping the en_GB "translation"?

**Comment by samhed** [comment]
I get the following when loading vnc.html while having my browser set to using English (United States):

> Failed to load resource: the server responded with a status of 404 (File not found) http://localhost:6080/app/locale/en.js

**Comment by DirectXMan12** [comment]
> Is there any reason to why you are keeping the en_GB "translation"?

So that proper British citizens can use the Queen's English, and not have to settle for using the improper form spoken in uppity former colonies! 😜 

Actually, I think we should just have a translation for every language, including the "default" one.

**Comment by ossman** [comment]
All comments should be fixed now.

File: core/util.js [hunk 1]
```diff
@@ -440,7 +440,81 @@ Util.Localisation = {
...
+                if (isAnyOf(elem.getAttribute("translate"), ["", "yes"])) {
+                    enabled = true;
+                } else if (isAnyOf(elem.getAttribute("translate"), ["no"])) {
+                    enabled = false;
+                }
+            }
+
+            if (enabled) {
+                if (elem.hasAttribute("abbr") &&
+                    elem.tagName === "th") {
```
Comment (CendioOssman) [inline core/util.js]: Damn. Missed one tag that needs to be uppercase. I need to fix this before we merge.

**Comment by samhed** [comment]
I see that you have written a wiki page about the translations as well, nice!

https://github.com/kanaka/noVNC/wiki/Translations

**Comment by samhed** [comment]
Note that the en_GB translation was removed.

**Pull request merged**